### ISSUE TITLE
Psuedocode Feedback 

### ISSUE THREAD

**Issue by klai22** [claire-j-wells/Deduper-claire-j-wells#2]
Hi Claire. 

Your pseudocode was highly organized and clear. I appreciated the context you provided ahead of the pseudocode in your .md file, specifically the part where you walk through the different cases your code will have to account for. Sorting by chromosomes alone at the start should be sufficient. 

No steps seem to be missing from your outline. However, a little more elaboration on the hamming distance calculations (and potentially turning it into a high level function) would have been more helpful. This aspect could certainly allow for more control over your algorithm and is something I didn’t think of beforehand! Additionally, specifying what exactly needs to be reset in step 4 could improve the clarity of your code’s goal. 

Your other proposed high level functions are reasonable. You may want to consider including more functions like one to extract the strandedness information for each read (and calculating hamming distance) . I liked your position calculator function! 

In conclusion, your pseudocode has an easy-to-follow flow and I am excited to see what the code will look like!
